build-image-src/Dockerfile-dotnet7

```diff
@@ -0,0 +1,89 @@
+ARG IMAGE_ARCH
+FROM public.ecr.aws/sam/lambda/dotnet7:latest-$IMAGE_ARCH
+
+# To learn more context around use of `amazonlinux:2` image please read comment in java11/build/Dockerfile
+# Copying root from runtimes image to AL2
+FROM public.ecr.aws/amazonlinux/amazonlinux:2
+COPY --from=0 / /rootfs
+
+# Installing by yum at copied location
+RUN yum groupinstall -y development --installroot=/rootfs && \
+  yum install -d1 --installroot=/rootfs -y \
+  yum \
+  tar \
+  gzip \
+  unzip \
+  python3 \
+  jq \
+  grep \
+  curl \
+  make \
+  rsync \
+  binutils \
+  gcc-c++ \
+  procps \
+  libgmp3-dev \
+  zlib1g-dev \
+  libmpc-devel \
+  python3-devel \
+  clang krb5-devel openssl-devel llvm libicu \
```

Comment:
nit: all other packages are in a new line (probably easier to read or catch what is been installed). Can we move all these packages into their own lines?